src/test/java/uk/gov/pay/api/it/validation/PaymentsResourceAmountValidationITest.java

```diff
@@ -48,6 +48,29 @@ public void createPayment_responseWith422_whenAmountIsNegative() throws IOExcept
                 .assertThat("$.description", is("Invalid attribute value: amount. Must be greater than or equal to 1"));
     }
 
+    @Test
+    public void createPayment_responseWith422_whenAmountIsLessThanMin() throws IOException {
+
+        String payload = "{" +
+                "  \"amount\" : 0," +
+                "  \"reference\" : \"Some reference\"," +
+                "  \"description\" : \"Some description\"," +
+                "  \"return_url\" : \"https://somewhere.gov.uk/rainbow/1\"" +
+                "}";
```

Comment:
Personally I'd prefer something like https://github.com/alphagov/pay-connector/blob/037141a1da79e72eb365ea950328668eaed7497a/src/test/java/uk/gov/pay/connector/it/resources/CreateGatewayAccountResourceTest.java#L49-L54 but the way it's done currently (by writing out the json explicitly) is still readable for now.

Reply:
Happy to make improvements but will keep them in separate PRs to avoid too much change within the tests we're relying on as we go. I'll look at this again when I come to clean up at the end if you don't mind.

Reply:
Yeah, no need to make the improvement now. I think most of these validation tests can be collapsed but it should be done separately from this PR.